docs/intro/contributing.txt

```diff
@@ -162,22 +165,23 @@ If the ``source`` command is not available, you can try using a dot instead:
 
 .. admonition:: For Windows users
 
-    To activate your virtualenv on Windows, run:
+    To activate your virtual environment on Windows, run:
 
     .. code-block:: doscon
 
         ...\> %HOMEPATH%\.virtualenvs\djangodev\Scripts\activate.bat
 
-You have to activate the virtualenv whenever you open a new terminal window.
-virtualenvwrapper__ is a useful tool for making this more convenient.
+You have to activate the virtual environment whenever you open a new
+terminal window. virtualenvwrapper__ is a useful tool for making this
+more convenient.
 
 __ https://virtualenvwrapper.readthedocs.io/en/latest/
 
 Anything you install through ``pip`` from now on will be installed in your new
```

Comment:
"from now on" -> "from now in this terminal" ?

Reply:
that's still not quite correct because you can `deactivate` or change PATH etc

Reply:
"whilst the virtual environment is active" ?

Reply:
done